src/averageAges.js

```diff
@@ -15,13 +17,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menList = people.filter(man => (
+    !century
+      ? man.sex === 'm'
```

Comment:
`man.sex === 'm'` - you are using this expression twice. So, let's create a variable for it.